x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -152,9 +156,13 @@ export const ActionForm = ({
     (async () => {
       try {
         setIsLoadingActionTypes(true);
-        const registeredActionTypes = (await loadActionTypes({ http, featureId })).sort((a, b) =>
-          a.name.localeCompare(b.name)
-        );
+        const registeredActionTypes = (
+          await loadActionTypes({
+            http,
+            featureId,
+            includeSystemActions: true,
```

Comment:
do we need to add a test here to make sure that we always do the call with `includeSystemActions` to be true.